.github/workflows/main.yml

```diff
@@ -23,9 +23,10 @@ jobs:
 
   test:
     name: Test Suite
+    if: ${{ !startsWith(github.event.pull_request.title, '[WIP]') }}
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        os: [ubuntu-latest, windows-latest] # macos-latest might be readded later
```

Comment:
If you want to include a comment, comment on the 1 to 10 ratio of minutes of macos. That's the interesting part.